CONTRIBUTING.md

```diff
@@ -5,7 +5,24 @@ pull requests. This document outlines some of the conventions on development
 workflow, commit message formatting, contact points and other resources to make
 it easier to get your contribution accepted.
 
-We gratefully welcome improvements to documentation as well as to code.
+We gratefully welcome improvements to issues and documentation as well as to code.
+
+## Working on issues
+
+If you like Flux and want to get involved in the project, a great way to get started
+is reviewing our [blocked-needs-validation](https://github.com/weaveworks/flux/issues?q=is%3Aissue+is%3Aopen+label%3Ablocked-needs-validation) issues.
+
+The idea here is that new issues are first confirmed by somebody in the team,
+which might require asking for more information, testing with a fresh Flux
+environment and once confirmed (and more information added to the issue), removing
+the `blocked-needs-validation` label. Afterwards the issue can be worked on and
```

Comment:
.. a fresh Flux environment. Once confirmed, the `blocked-needs-validation` label is removed, and the issue can be worked on ...